advanced/Templates/pihole-FTL.service

```diff
@@ -23,27 +20,18 @@ start() {
     echo "pihole-FTL is already running"
   else
     # Touch files to ensure they exist (create if non-existing, preserve if existing)
-    touch /var/log/pihole-FTL.log /var/log/pihole.log
-    touch /run/pihole-FTL.pid /run/pihole-FTL.port
-    touch /etc/pihole/dhcp.leases
     mkdir -p /run/pihole
-    mkdir -p /var/log/pihole
-    chown pihole:pihole /run/pihole /var/log/pihole
-    # Remove possible leftovers from previous pihole-FTL processes
-    rm -f /dev/shm/FTL-* 2> /dev/null
-    rm /run/pihole/FTL.sock 2> /dev/null
+    touch /run/pihole-FTL.pid /run/pihole-FTL.port /var/log/pihole-FTL.log /var/log/pihole.log /etc/pihole/dhcp.leases
     # Ensure that permissions are set so that pihole-FTL can edit all necessary files
-    chown pihole:pihole /run/pihole-FTL.pid /run/pihole-FTL.port
```

Comment:
In your current version of the PR, `/run/pihole-FTL.pid /run/pihole-FTL.port` are not `chown`ed any more.

As these two files are only `touch`ed by `root` (two lines above) + `chmod`ed to `0644` (some lines below), new installations will have permission issues.

Is this intended?

Reply:
Below, after touched, `chown -R pihole:pihole /run/pihole ...` changes owner/group recursively for this directory.

Reply:
Okay it's now these files `/run/pihole-FTL.pid /run/pihole-FTL.port /var/log/pihole-FTL.log /var/log/pihole.log /etc/pihole/dhcp.leases` first touched, then chowned, then chmoded, and `/run/pihole /etc/pihole` additionally chowned to allow FTL create the socket and the database files (when missing). The database file ) need to be chowned separately as they may not exist so that "-f" can be passed for those. All others must exist, so we want to see an error message when touch failed somehow or any other issue made those disappear.

Reply:
We could easily move the other two files, I guess? Especially because the `pid` file seems to be used nowhere at all. Furthermore, the `port` file only contains the Telnet port 4711 (or whatever non-custom might have been set in `pihole-FTL.conf`) and will be removed in v6.0.

Reply:
The port file is used by chronometer and PADD:
- https://github.com/pi-hole/PADD/blob/00126a9/padd.sh#L124
- https://github.com/pi-hole/pi-hole/blob/7c0c30f/advanced/Scripts/chronometer.sh#L17

So those would need to be updated before removing the port file.